projects/b2stage/backend/apis/ingestion.py

```diff
@@ -11,7 +11,7 @@
 from restapi.services.uploader import Uploader
 from restapi.flask_ext.flask_celery import CeleryExt
 from b2stage.apis.commons.cluster import ClusterContainerEndpoint
-from b2stage.apis.commons.cluster import INGESTION_DIR, MOUNTPOINT
+from b2stage.apis.commons.cluster import INGESTION_DIR, LOCALPATH
```

Comment:
LOCALPATH is not intended to be a path on the host?

LOCALPATH on the host is bound to MOUNTPOINT into the container, HTTP-api should not be able to see LOCALPATH

Reply:
You're totally right. My bad. The problem was elsewhere and I solved it with your advice. I'm closing this issue.